idd/Energy+.idd.in

```diff
@@ -67194,6 +67249,33 @@ AirLoopHVAC:ReturnPath,
        \type object-list
        \object-list ReturnPathComponentNames
 
+AirLoopHVAC:ExhaustSystem,
+       \memo Defines a general exhaust systems with a central exhaust fan drawing from one or more
+       \memo ZoneHVAC:ExhaustControl outlet nodes.
+  A1 , \field Name
+       \required-field
+       \note Name of the exhaust system
+  A2 , \field Availability Schedule Name
+       \note Enter the name of the Availability Schedule for the Exhaust System.
+       \note If this field is blank, the exhaust system is always available.
+       \type object-list
+       \object-list ScheduleNames
+  A3 , \field AirLoopHVAC:ZoneMixer Name
```

Comment:
\field Zone Mixer Name
To be consistent with other similar fields (e.g. AirTerminal:SingleDuct:ConstantVolume:FourPipeInduction).

Reply:
Text updated.